src/WordPress_Object/Clarkson_Object.php

```diff
@@ -148,21 +148,23 @@ public function get_parent() {
 	 *
 	 * See issue: https://github.com/level-level/Clarkson-Core/issues/121
 	 *
-	 * @return array Children
+	 * @return Clarkson_Object[] Children
 	 */
 	public function get_children() {
-		return get_children( 'post_parent=' . $this->get_id() );
+		$children = get_children( 'post_parent=' . $this->get_id() );
+		return Objects::get_instance()->get_objects( $children );
 	}
```

Comment:
Is het handig om de $args (als array) parameter beschikbaar te stellen op `get_children()`? Daarmee wordt het geheel wat flexibeleren. https://developer.wordpress.org/reference/functions/get_children/

Reply:
Resolved in https://github.com/level-level/Clarkson-Core/pull/208/commits/365b85dd5000a87781031ac16f6519c581f50ff6